src/views/Order.vue

```diff
@@ -44,34 +47,35 @@
                 <ion-icon slot="end" :icon="informationCircleOutline" />
               </ion-item>
             </ion-list>
-          </div>
+          </div> -->
 
           <div class="info">
             <ion-card>
               <ion-list>
                 <ion-item lines="none">
-                  <ion-label>John Doe</ion-label>
-                  <ion-chip slot="end">
+                  <ion-label> {{ order.customer?.name }} </ion-label>
+                  <!-- TODO: handle this property to display loyalty options -->
+                  <ion-chip slot="end" v-if="order.cusotmerLoyaltyOptions">
```

Comment:
I think this should be handled with a filter, check preorder app for reference

Reply:
Added a filter to correctly handle the customerLoyaltyOptions.